viz2code container build fail

viz2code build fail

```sh
Step 30/33 : RUN npm run build
 ---> Running in 27be72bd74e7

> frontend@0.1.0 build
> react-scripts build

Creating an optimized production build...
One of your dependencies, babel-preset-react-app, is importing the
"@babel/plugin-proposal-private-property-in-object" package without
declaring it in its dependencies. This is currently working because
"@babel/plugin-proposal-private-property-in-object" is already in your
node_modules folder for unrelated reasons, but it may break at any time.

babel-preset-react-app is part of the create-react-app project, which
is not maintianed anymore. It is thus unlikely that this bug will
ever be fixed. Add "@babel/plugin-proposal-private-property-in-object" to
your devDependencies to work around this error. This will make this message
go away.
  
Failed to compile.

[eslint] package.json » eslint-config-react-app/jest#overrides[0]:
	Environment key "jest/globals" is unknown


npm ERR! code 1
npm ERR! path /visualization/frontend
npm ERR! command failed
npm ERR! command sh -c react-scripts build

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2024-03-08T06_30_51_574Z-debug.log
The command '/bin/sh -c npm run build' returned a non-zero code: 1
ERROR: Service 'viz2code' failed to build : Build failed
```

--- a/visualization/frontend/package.json
+++ b/visualization/frontend/package.json
@@ -39,8 +39,7 @@
   },
   "eslintConfig": {
     "extends": [
-      "react-app",
-      "react-app/jest"
+      "react-app"
     ]
   },
   "browserslist": {

